src/main/resources/stsjorbsmodResources/localization/eng/JorbsMod-UI-Strings.json

```diff
@@ -71,6 +71,18 @@
       "At the end of turn 7, #ySnap. NL (It is turn #b%1$s now.)"
     ]
   },
+  "stsjorbsmod:ThirstOption": {
+    "TEXT": [
+      "Thirst",
+      "Consume wrath from an attack, heal 5 hp, and gain 1 max hp per wrath",
```

Comment:
`#yWrath`? (not sure if smart text works here)

Reply:
If smart test works, I also recommend prepending `#b` to the numbers.

Reply:
Smart text doesn't work, unfortunately